application_examples/google_Colaboratory/README.md

```diff
@@ -0,0 +1,13 @@
+# Example script: run AgentQL script in online development environment
+
+This example demonstrates how to run AgentQL script in online development environment like [Google Colaboratory](https://colab.research.google.com/).
+
+## Run the script
+
+- Go to [Google Colaboratory](https://colab.research.google.com/) website
+- Upload the **agentql.ipynb** file to Google Colab
+- Run the script by clicking the run button of each cell sequentially.
+
+## Play with the query
+
+Install the [AgentQL Chrome DevTools extension](https://docs.agentql.com/docs/installation/chrome-extension-installation/) to play with the AgentQL query. [Learn more about the AgentQL query language](https://docs.agentql.com/docs/agentql-query/query-intro)
```

Comment:
@frankfeng98 can I ask you to update it to "AgentQL Debugger Chrome extension" for all examples, pls?